package.json

```diff
@@ -17,41 +17,57 @@
     "test": "npm run lint",
     "test-only": "echo \"Error: no test specified\" && exit 1",
     "build": "rm -rf lib/* && cp src/*.css lib/ && babel src --optional runtime --stage 0 --out-dir lib",
-    "start-website": "webpack-dev-server  --config webpack.examples.config.js"
+    "start-website": "webpack-dev-server  --config webpack.examples.config.js",
+    "deploy-website": "gh-pages-deploy"
+  },
+  "pre-commit": [
+    "lint",
+    "build"
+  ],
+  "gh-pages-deploy": {
+    "staticpath": "website",
+    "prep": [
+      "build-website"
+    ]
   },
   "license": "BSD-3-Clause-LBNL",
   "dependencies": {
     "pondjs": "^0.3.x",
     "babel-runtime": "~5.8.3",
     "d3": "^3.5.5",
+    "invariant": "^2.1.1",
+    "jquery": "^2.1.4",
```

Comment:
Hmmm... that might have come is with a merge. I don't think this code should depend on jQuery anymore. Let me think about that.